src/Common/tests/Performance/PerfUtils.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.IO;
+
+namespace System
+{
+    /// <summary>Static helper class for performance tests</summary>
+    public static class PerfUtils
+    {
+        private static Random rand = new Random(1234132);
```

Comment:
They can not run concurrently, no. I'll check up to make sure this is still the case, but last time I checked the Benchmark class made use of Windows ETW tracing which concurrent testing would interfere with.